src/common.js

```diff
@@ -259,21 +261,24 @@ function wrap(cmd, fn, options) {
     try {
       var args = [].slice.call(arguments, 0);
 
+      // Log the command to stderr, if appropriate
       if (config.verbose) {
-        args.unshift(cmd);
-        console.error.apply(console, args);
-        args.shift();
+        console.error.apply(console, [cmd].concat(args));
       }
 
-      if (options && options.notUnix) {
+      if (options && options.notUnix) { // this branch is for exec()
         retValue = fn.apply(this, args);
-      } else {
+      } else { // and this branch is for everything else
         if (args[0] instanceof Object && args[0].constructor.name === 'Object') {
-          args = args; // object count as options
+          args = args; // a no-op, allowing the syntax `touch({'-r': file}, ...)`
```

Comment:
Ok, I'd say that we should do something like (instead of `args = args;`, `// pass` or `// noop`, to indicate that nothing's happening).